### DISCUSSION TITLE
posts/quantumult-x-chain-proxy-setup/

### DISCUSSION THREAD

**Discussion by giscus** [WincerChan/blog-comments#5] [Announcements]
# posts/quantumult-x-chain-proxy-setup/

最近因为 Clash for Windows 作者被请喝茶的缘故，Clash 从内核到各平台客户端的仓库大部分都删库或者归档了，这也不能怪开发者太过风声鹤唳，毕竟还身在国内，写开源而已，犯不上和自己的人身安全作对。之前我一直都是使用 Clash 作为主要的科学上网工具，不过经此一役后我也在考虑是否应该放弃 Clash，于是我仔细梳理了一下目前对科学上网的需求，最终决定将 Clash 切换成 Quantumult X。

https://blog.itswincer.com/posts/quantumult-x-chain-proxy-setup/

**Comment by tchouhouanyu** [comment]
终于有大佬普及QX的链式代理了！大佬可以出一期视频吗？摸索了半天还是不会。

**Reply by WincerChan** [reply]
这两天有空我出一个图片版教程

**Comment by tchouhouanyu** [comment]
谢谢大佬！

LoeRfy ***@***.***>于2024年1月27日 周六19:59写道：

**Comment by JasonChou0v0** [comment]
博主的图文教程有了没qvq，刚好在研究代理链有不懂的地方想咨询下。通过使用解析器将订阅的一个节点转向另一个节点，但是在检查是否成功的时候好像发现都走的同一个，自己转自己？

**Reply by WincerChan** [reply]
前两天有些忙，图片教程已经上传。不过用 quanx 的解析器来解析链式代理我也没玩明白

**Comment by WincerChan** [comment]
图文教程：
![配置教程](https://github.com/WincerChan/blog-comments/assets/16951341/9350fe5f-e8a7-4255-926f-e0ce00fed7ab)

**Reply by JasonChou0v0** [reply]
你好 非常感谢你的及时回复，不过我看了图文教程有一些想法，同时也想请教一下你（没有自己的vps 拿出单条节点用作中专，但是每次换中专ip 比较麻烦。
我在研究解析器的代理链时，我参考的教程：https://t.me/QuanX_API/133   感觉比较绕，没理解明白
1、分流里面引用
http://api.example.com/subscribe?token=2e142c4524#relay=Agent%20Chain
偏好走Agent Chain
2.添加分流
https://raw.githubusercontent.com/blackmatrix7/ios_rule_script/master/rule/Surge/GoogleSearch/GoogleSearch.list#via=0
偏好走Youtube
然后是这样
![image](https://github.com/WincerChan/blog-comments/assets/76592626/df9e2c29-b7a2-4691-b74a-240729b04eec)
能帮忙分析下嘛，TG被禁止发起私聊，方便的话可以[联系我](https://t.me/JasonChou_T)

**Reply by JasonChou0v0** [reply]
补充
![image](https://github.com/WincerChan/blog-comments/assets/76592626/8a9a5e4b-bad0-4b41-b986-a5556333c563)
![image](https://github.com/WincerChan/blog-comments/assets/76592626/6c96b31e-f184-4dcf-805f-fcc727a42b94)

**Reply by WincerChan** [reply]
这个教程我也看过，但是我按照这个教程没有成功，就没看这个了

**Reply by JasonChou0v0** [reply]
资源引用后看具体的分流设置好像和你的一样，不过补的两张图我不确定是否已经成功（我感觉没有T_T
